app/konnect/index.md

```diff
@@ -10,7 +10,7 @@ breadcrumb: Overview
 management platform designed from the ground up for the cloud native era
 and delivered as a service. This platform lets you build modern applications
 better, faster, and more securely. The control plane is hosted
-in the cloud by Kong, while the data plane is managed by you within your
+in the cloud by Kong, while the data plane is managed by you or Kong in your
```

Comment:
Same as in the doc above. I don't think its powerful to say managed by you 'or kong'. I think highlighting choice and flexibility is the most important part.